### PR TITLE
Tx sender

### PR DESCRIPTION
**Pull request by forgxyz**
# Description

_Please include a summary of changes and related issue (if any)._
Adds tx_sender to the transactions models.
A full refresh _is_ required for the transactions models.

# Tests

- [x] Please provide evidence of your successful `dbt run` / `dbt test` here
- [x] Any comparison between `prod` and `dev` for any schema change
```
➜  terra_dbt git:(tx_sender) dbt run -s silver__transactions+ --full-refresh
02:08:53  Running with dbt=1.2.0
02:08:54  Found 43 models, 92 tests, 0 snapshots, 0 analyses, 680 macros, 1 operation, 0 seed files, 43 sources, 0 exposures, 0 metrics
02:08:54
02:08:59
02:08:59  Running 1 on-run-start hook
02:08:59  1 of 1 START hook: terra.on-run-start.0 ........................................ [RUN]
02:08:59  1 of 1 OK hook: terra.on-run-start.0 ........................................... [OK in 0.00s]
02:08:59
02:08:59  Concurrency: 4 threads (target='dev')
02:08:59
02:08:59  1 of 2 START incremental model silver.transactions ............................. [RUN]
02:09:23  1 of 2 OK created incremental model silver.transactions ........................ [SUCCESS 1 in 24.49s]
02:09:23  2 of 2 START view model core.fact_transactions ................................. [RUN]
02:09:26  2 of 2 OK created view model core.fact_transactions ............................ [SUCCESS 1 in 2.84s]
02:09:26
02:09:26  Finished running 1 incremental model, 1 view model, 1 hook in 0 hours 0 minutes and 32.74 seconds (32.74s).
02:09:26
02:09:26  Completed successfully
02:09:26
02:09:26  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
```

The following test failure is known and reported.
```
02:12:32  Completed with 1 error and 0 warnings:
02:12:32
02:12:32  Failure in test tx_gaps_core__fact_transactions_block_id__TX_ID__tx_count (models/core/core__fact_transactions.yml)
02:12:32    Got 71 results, configured to fail if != 0
02:12:32
02:12:32    compiled SQL at target/compiled/terra/models/core/core__fact_transactions.yml/tx_gaps_core__fact_transactions_block_id__TX_ID__tx_count.sql
02:12:32
02:12:32    See test failures:
  ---------------------------------------------------------------
  select * from TERRA_DEV.tx_gaps_core.fact_transactions_block_id
  ---------------------------------------------------------------
02:12:32
02:12:32  Done. PASS=19 WARN=0 ERROR=1 SKIP=0 TOTAL=20
```

# Checklist
- [x] Follow [dbt style guide](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md)
- [x] Tag the person(s) responsible for reviewing proposed changes
- [x] Notes to deployment, if a `full-refresh` is needed for any table
- [x] Run `git merge main` to pull any changes from remote into your branch prior to merge.

### CODE REVIEW

**Comment by forgxyz** [comment]
^ no
turning preview deployments off, but thanks anyway Vercel

File: models/silver/silver__transactions.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ models:
       - name: authorizer_public_key
         description: "{{ doc('authorizer_public_key') }}"
 
+      - name: tx_sender
+        description: "{{ doc('tx_sender') }}"
+
```
Comment (D0R11) [inline models/silver/silver__transactions.yml:26]: Not sure why there are tests in the core yml but not here

Comment (forgxyz) [reply models/silver/silver__transactions.yml:26]: No need to have the same test on core & silver - it would just be running the same thing twice

Comment (D0R11) [reply models/silver/silver__transactions.yml:26]: makes sense

**Review by D0R11 — APPROVED** [review]

**Pull request merged**